src/AppBanner/index.test.tsx

```diff
@@ -90,4 +90,49 @@ describe('AppBanner', () => {
         // Nothing rendered as the imageUrl was not allowed
         expect(container.firstChild).toBeNull();
     });
+
+    it('does not render the banner when no body text provided', () => {
+        const { brazeMessageProps, ...props } = baseProps();
+        const invalidMessageProps = {
+            ...brazeMessageProps,
+            body: undefined,
+        };
+
+        const { container } = render(
+            <AppBanner {...props} brazeMessageProps={invalidMessageProps} />,
+        );
+
+        // Nothing rendered as the imageUrl was not allowed
```

Comment:
I don't think this comment is accurate 😃  - probably copy pasted from another test (also exists in other tests below)